cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/KafkaCruiseControl.java

```diff
@@ -666,4 +669,16 @@ private void sanityCheckHardGoalPresence(List<String> goals, boolean skipHardGoa
       }
     }
   }
+
+  /**
+   * Sanity check whether the provided brokers to be decommissioned/demoted exist in cluster or not.
```

Comment:
Does this method care whether the provided brokers will be decommissioned or demoted?

`Sanity check whether the provided brokers to be decommissioned/demoted exist in cluster or not.` -> `Sanity check whether the provided brokers exist in cluster or not.`?